src/App.tsx

```diff
@@ -1,61 +1,130 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { TodoItem } from './types/TodoItems';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUserById(userId: number): User | null {
+  return usersFromServer.find((user: User) => user.id === userId) || null;
+}
+
+function getTodoId(todos: TodoItem[]): number {
+  if (todos.length === 0) {
+    return 1;
+  }
+
+  const maxId = Math.max(...todos.map(todo => todo.id));
+
+  return maxId + 1;
+}
+
+export const App: React.FC = () => {
+  const [todo, setTodo] = useState<TodoItem[]>(
+    todosFromServer.map(todos => ({
+      ...todos,
+      user: getUserById(todos.userId),
+    })),
+  );
```

Comment:
I think it would be better to get the value before and use the name 'todos' instead of 'todo'